src/pages/StarterTips/page-6.vue

```diff
@@ -1,10 +1,10 @@
 <script setup lang="ts">
-const dots = [1, 2, 3, 4]
+const dots = [1, 2, 3, 4, 5, 6]
```

Comment:
Move this to utils or constants, because it's repeating

Reply:
later